nov.el

```diff
@@ -510,7 +510,8 @@ the HTML is rendered with `shr-render-region'."
          (directory (file-name-directory current-path))
          (path (file-truename (nov-make-path directory filename)))
          (index (nov-find-document (lambda (doc)
-                                     (string-suffix-p path (cdr doc))))))
+                                     (equal (file-attributes (file-truename (cdr doc)))
+                                            (file-attributes (file-truename path)))))))
```

Comment:
No need to use `file-truename` again (it has been used a few lines above already).